index.html

```diff
@@ -397,6 +398,7 @@
 D_\mathrm{GGX}(m) \propto \left( 1 + \tan^2\theta_m \left(\frac{\cos^2\phi_m}{\alpha_t^2} + \frac{\sin^2\phi_m}{\alpha_b^2}\right) \right)^{-2}
 \end{equation}
 which reduces to the isotropic form when $\alpha_t = \alpha_b = \alpha$.
+Efficient techniques for sampling BSDFs employing the anisotropic GGX microfacet model are presented in [#Heitz2014a].
```

Comment:
This is a nice addition. However the cited paper is no longer the state of the art. I would suggest replacing this citation with these two instead:

`[#Heitz2018]: Eric Heitz. *Sampling the GGX Distribution of Visible Normals*, Journal of Computer Graphics Techniques (2018).`

`[#Dupuy2023]: Jonathan Dupuy, Anis Benyoub. *Sampling Visible GGX Normals with Spherical Caps*, High-Performance Graphics (2023).`

Reply:
Good point, @peterkutz, and indeed it's the Depuy/Benyoub paper that we currently use in MaterialX GLSL/MSL.

https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/libraries/pbrlib/genglsl/lib/mx_microfacet_specular.glsl#L64

Reply:
Added refences to both in https://github.com/AcademySoftwareFoundation/OpenPBR/pull/171/commits/edc8216c5e6e672763f4e930cf09ae56358cd130.